charts/schema-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,35 @@
+# /**
+#  * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License");
+#  * you may not use this file except in compliance with the License.
+#  * You may obtain a copy of the License at
+#  *
+#  *     http://www.apache.org/licenses/LICENSE-2.0
+#  */
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "schema-registry.fullname" . }}-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "schema-registry.commonLabels" . | indent 4 }}
+data:
+  "CONTROLLER_URL": "{{ .Values.controllerUri }}"
+  "STORE_TYPE": "{{ .Values.storeType }}"
+  "AUTHORIZATION_ENABLED": "{{ .Values.authentication.enabled }}"
+  "TLS_ENABLED": "{{ .Values.tls.enabled }}"
+  "pravega_client_auth_loadDynamic": "{{ .Values.client.loadDynamic }}"
+  "pravega_client_auth_method": "{{ .Values.client.method }}"
+  "pravega_client_auth_token": "{{ .Values.client.token }}"
+  {{- if .Values.authentication.enabled }}
+  "USER_PASSWORD_FILE": "/etc/auth-passwd-volume/{{ .Values.authentication.userPasswordFile }}"
```

Comment:
Is this correct? Isn't it `/etc/secret-volume`?

Reply:
`/etc/auth-passwd-volume` is where the auth resources are mounted and `/etc/secret-volume` is where the tls resources are mounted

Reply:
I realized later this is alright.